### PR TITLE
Fixed inconsistent return value in validate()

### PR DESCRIPTION
**Pull request by mintalicious**
Validator::validate() should return true if the validation is successful, i.e. there were no errors.

### CODE REVIEW

**Comment by pdscopes** [comment]
Hi @mintalicious, thank you for your contribution!

I love that you've translated to German and added a new `contains` rule.

Please can you split this into two merge requests as, while I agree `Validator::validate()` would make more sense semantically if it returned `true` if the tests passed, it would be a breaking change and so would need to be part of a new major version.

Further, the corrections to change `%value` to `:value` aren't correct as the `Validator::getProcessedErrors()` function converts the placeholders marked with a colon `:` to a pretty attribute but those marked with a percentage `%` are taken raw.

In summary, please can you:
1.  Revert the placeholder changes from `:value` back to `%value`
2. Split this into 2 merge requests, one with the German translation and contains, the other with the backwards incompatible change for `Validator::validate`.

Thanks again!
